pubsub/README.rst

```diff
@@ -90,15 +90,16 @@ the topic, and subscribe to that.
     from google.cloud import pubsub
 
     subscriber = pubsub.SubscriberClient()
-    topic = 'projects/{project_id}/topics/{topic}'.format(
+    topic_name = 'projects/{project_id}/topics/{topic}'.format(
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         topic='MY_TOPIC_NAME',  # Set this to something appropriate.
     )
     subscription_name = 'projects/{project_id}/subscriptions/{sub}'.format(
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         sub='MY_SUBSCRIPTION_NAME',  # Set this to something appropriate.
     )
-    subscription = subscriber.create_subscription(topic, subscription)
+    subscription = subscriber.create_subscription(
+        name=subscription_name, topic=topic_name)
```

Comment:
My preference would be to leave the names out unless these are kwargs in the method itself, though I know you feel differently @tseaver